tests/charm++/zerocopy/Makefile

```diff
@@ -6,9 +6,15 @@ DIRS = \
   dereg_and_nodereg \
   bcast_nonzero_root \
   pup_buffer \
+  large_p2p \
+
+NONSCALEDIRS = \
+  large_p2p \
 
 TESTDIRS = $(DIRS)
 
+TESTPDIRS = $(filter-out $(NONSCALEDIRS),$(TESTDIRS))
```

Comment:
The `testp` rule below should have `TESTDIRS` changed to `TESTPDIRS`.

Reply:
Done.